Lesson 2

exnk opened the pull request:
Вроде как все даже и готово, на всякий случай где нужно добавил обработчики исключений, чтобы небыло соблазна вводить в переменную возраст значение "картошка"

exnk committed:
Lesson 1 Ходкевич А

Вроде как все даже и готово, на всякий случай где нужно добавил обработчики исключений, чтобы небыло соблазна вводить в переменную имя значение "картошка"

exnk committed:
Добавил свои ФИО -__-

в первый раз забыл добавить ФИО

manchenkoff commented on lesson01/home_work/hw01_easy.py:
```diff
@@ -18,7 +33,29 @@
+	print(new)
+
+
+task2(a=input('Введите значение А:    '), b=input('Введите значение B:    '))
```
использование input() в аргументах нежелательно с точки зрения правил написания кода, но пока прощается)

manchenkoff commented on lesson01/home_work/hw01_easy.py:
```diff
@@ -18,7 +33,29 @@
+			print('Ваш возраст меньше 18, попробуйте войти позднее')
+
+
+task3(age=input('Введите ваш возраст:    '))
```
лучше выносить проверку на числа вне функции, оставляя только тот функционал, для которого она будет служить

manchenkoff commented on lesson01/home_work/hw01_normal.py:
```diff
@@ -11,6 +11,18 @@
+	except ValueError:
+		print('Это не число!!!')
+	else:
+		lst = list(map(int, str(number)))
```
хороший способ

exnk committed:
Lesson 2

exnk committed:
Lesson 2

Доделал задание про башню. Единственное, из за использования словарей оч долго отрабатывает... иного не придумал...

exnk committed:
Merge commit '7172ebd'

Доделал 3 хард

exnk committed:
Lesson 2

Немного доработал задачу 3 хард

exnk committed:
Lesson2

Поправил код в соотв с PEP8 за исключением наименования переменной lst

exnk committed:
Merge commit '60025e8'

upd

exnk committed:
Lesson 2

Проблема с ветками вроде решилась 3=

exnk committed:
Lesson 2

Проблема с ветками вроде решилась 3=

exnk committed:
Lesson 2

Last upd
